### ISSUE TITLE
Improve the website

### ISSUE THREAD

**Issue by navendu-pottekkat** [learningOrchestra/learningOrchestra#92]
The landing page of learningOrchestra: https://github.com/learningOrchestra/learningOrchestra.com, can be improved by-

- Changing the design
- Adding more info
- And anything else that would help

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat I never thought that learningOrchestra will get this, this is so, so cool!

**Comment by riibeirogabriel** [comment]
I will provide a domain

**Comment by riibeirogabriel** [comment]
and thanks!

**Comment by navendu-pottekkat** [comment]
> I will provide a domain

You can add the custom domain and add docs subdomain to  our docs

**Comment by riibeirogabriel** [comment]
Yes, i will make this, yet in this week.

**Comment by LaChapeliere** [comment]
What about a ReadTheDoc setup? The design wouldn't be very specific but it's made for dev documentation

**Comment by navendu-pottekkat** [comment]
> What about a ReadTheDoc setup? The design wouldn't be very specific but it's made for dev documentation

We have a readthedoc setup already, there is also a separate landing page for the project https://github.com/learningOrchestra/learningOrchestra.com

**Comment by LaChapeliere** [comment]
> > What about a ReadTheDoc setup? The design wouldn't be very specific but it's made for dev documentation
> 
> We have a readthedoc setup already, there is also a separate landing page for the project https://github.com/learningOrchestra/learningOrchestra.com

Cool, my bad? Are you serving it anywhere yet? Because when following the link to the doc, I ended up in a repo with markdown files, not on a documentation page

**Comment by navendu-pottekkat** [comment]
This is the landing page: [Landing page](https://learningorchestra.github.io/learningOrchestra.com/)
[Landing page repo](https://github.com/learningOrchestra/learningOrchestra.com)

This is the doc page: [Doc page](https://learningorchestra.github.io/learningOrchestra-docs/)
[Docs repo](https://github.com/learningOrchestra/learningOrchestra-docs)

[Python Client repo](https://github.com/learningOrchestra/learningOrchestra-python-client)

**Comment by LaChapeliere** [comment]
Ah, yes, cool webpage! I think the links in the Readme are broken actually, that's why I didn't end up on those, the only link I found that works was to the docs repos.

**Comment by navendu-pottekkat** [comment]
> Ah, yes, cool webpage! I think the links in the Readme are broken actually, that's why I didn't end up on those, the only link I found that works was to the docs repos.

Yes they are a work in progress, we moved to an organisation account from @riibeirogabriel 's personal account and there are some loose ends that we need to fix. 

We will fix them before we promote the repo in the coming weeks. 

@LaChapeliere You will check and fix the docs right?

**Comment by LaChapeliere** [comment]
> @LaChapeliere You will check and fix the docs right?

Can do. It'll take me some time to reach that part though

**Comment by LaChapeliere** [comment]
Found this: http://opencobra.github.io/
I think the design's awesome (maybe not completely accessible, I haven't checked the accessibility features) and @riibeirogabriel might also find some structure or feature ideas in their packages

**Comment by navendu-pottekkat** [comment]
looks good, maybe we can add it on top of our current website
